### DISCUSSION TITLE
Share to social networking not get preview

### DISCUSSION THREAD

**Discussion by jimmyliao** [Lednerb/bilberry-hugo-theme#520] [General]
<img width="492" alt="image" src="https://user-images.githubusercontent.com/151156/225316019-63078675-9f57-47ed-889f-fea13757417a.png">

**Comment by jimmyliao** [comment]
LinkedIn
<img width="565" alt="image" src="https://user-images.githubusercontent.com/151156/225316901-318d09c1-e1fc-4b45-8fa5-fa548661ecf3.png">

**Comment by igor-baiborodine** [comment]
@jimmyliao, It needs to be clarified what the issue is.

**Comment by Sp-cy** [comment]
any image in the page bundle called featuredImage will be used as social media preview image. You can also set it up in the front matter.
